helpers/configureImage.js

```diff
@@ -0,0 +1,45 @@
+'use strict';
+const URL = require('url').URL;
+const utils = require('handlebars-utils');
+const common = require('./lib/common.js');
+
+const factory = () => {
+    return function(image, options) {
+        if (!utils.isObject(image) || !utils.isString(image.data) || !common.isValidURL(image.data)) {
+            // Return empty string if this does not appear to be an image object.
+            return ''
+        }
+
+        // 'https://cdn0.bigcommerce.com/s-hash/images/stencil/{:size}/l/some-image.original.jpg?c=2'
+        let url = new URL(image.data);
+        
+        //remove AIM query strings if already set by the application
+        url.searchParams.delete('imbypass');
```

Comment:
why are we ignoring them ? If they exists we should respect them and return early.

Reply:
The idea is that we give the theme developer more control. If they want to modify what we return, they're allowed to. They're already using `{{replace}}` for this, I wanted to provide a nicer interface.